app/src/utils/index.ts

```diff
@@ -0,0 +1,21 @@
+import { IError, isIErrorList } from 'src/types/validation';
+
+export const identityFunction = <T>(x: T) => x;
```

Comment:
Hva er greia med denne, og hvorfor bruker vi den? 🤔

Reply:
Den brukes når `toWriteModel` eller liknende ikkje "trengs" fordi `writeModel` for tida er lik `domainModel`. Kunne selvfølgelig latt vere å definere `toWriteModel` i slike tilfeller, men synes det er god hygiene å gjere det likevel, for sjølv om modellane tilfeldigvis er like no, er dei konseptuelt ulike - og vil vere ulike i framtida.